charts/aws-fsx-csi-driver/templates/controller-deployment.yaml

```diff
@@ -8,12 +8,10 @@ spec:
   replicas: {{ .Values.controller.replicaCount }}
   selector:
     matchLabels:
-      app: fsx-csi-controller
```

Comment:
we should keep this, it's maybe redundant with the new `part-of` label but nonzero users have probably come to depend on it existing (personally i find it useful for kubectl get)

also EBS still has it so my top priority is staying consistent to avoid more divergence! https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/templates/controller.yaml#L16
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/974